loadtest/store/memstore/store.go

```diff
@@ -615,7 +722,17 @@ func (s *MemStore) SetStatus(userId string, status *model.Status) error {
 		return errors.New("memstore: status should not be nil")
 	}
 
-	s.statuses[userId] = status
+	if userId != status.UserId {
```

Comment:
Wondering if this check relevant to this PR ? Or just something we did while we are here. No issues either ways, just checking.

Reply:
Yeah, nothing to do with the PR, just helps keeping consistent data.